census_view.cpp

```diff
@@ -1889,6 +2976,205 @@ void CensusView::UponPasteCensus(wxCommandEvent&)
     LMI_ASSERT(!class_parms().empty());
 }
 
+void CensusGridView::UponPasteCensus(wxCommandEvent&)
+{
+    auto const census_data = ClipboardEx::GetText();
```

Comment:
```suggestion
    std::string const census_data = ClipboardEx::GetText();
```
Again, just to avoid an unnecessary diff.